FastBank.Services/CustomerService/CustomerService.cs

```diff
@@ -40,37 +40,57 @@ public void Add(string name, string email, DateTime birthday, string password, R
             {
                 Add(customer);
             }
-
         }
 
         public List<string> ValidatеCustomer(Customer customer)
         {
             var validationErrors = new List<string>();
             CustomerExist(customer, validationErrors);
+
             //TODO validate password 
             //TODO validate email
-            //TODO validate age to be more then 18 and less then 100
+            CustomerAgeIsValid(customer, validationErrors);
             //TODO validate role
             return validationErrors;
         }
 
         public List<string> CustomerExist(Customer customer, List<string> validationErrors)
         {
-            var customers = _customerRepo.GetAll(); //TODO use IQueryable
+            var customers = _customerRepo.GetByEmail(customer.Email);
```

Comment:
Customers??
If you expect only one customer, variable name should be - customer

Reply:
I've modified it